### PR TITLE
Simplify advanced-tensorflow example

### PR DESCRIPTION
**Pull request by charlesbvll**
### What does this implement/fix? Explain your changes.

The default values of the advanced TensorFlow example make it hard to run on any hardware, this PR aims to make it easier.

### CODE REVIEW

File: examples/advanced_tensorflow/README.md [hunk 1]
```diff
@@ -44,14 +43,16 @@ If you don't see any errors you're good to go!
 
 ## Run Federated Learning with TensorFlow/Keras and Flower
 
-The included `run.sh` will call a script to generate certificates (which will be used by server and clients), start the Flower server (using `server.py`), sleep for 2 seconds to ensure the the server is up, and then start 10 Flower clients (using `client.py`). You can simply start everything in a terminal as follows:
+The included `run.sh` will call a script to generate certificates (which will be used by server and clients), start the Flower server (using `server.py`), sleep for 2 seconds to ensure the the server is up, and then start 2 Flower clients (using `client.py`). You can simply start everything in a terminal as follows:
```
Comment (danieljanes) [inline examples/advanced_tensorflow/README.md:46]: Can we go back to ten clients here? One part of the advanced example is to show some of the things one would do when running with more than two clients. The performance of the system is largely determined by how many clients are selected during a single round, not so much by how many clients there are in total.

**Review by danieljanes — DISMISSED** [review]

File: examples/advanced_tensorflow/README.md (hunk 1 above)
Comment (charlesbvll) [reply examples/advanced_tensorflow/README.md]: @danieljanes It indeed works with 10 clients, I changed it back!

File: examples/advanced_tensorflow/client.py [hunk 2]
```diff
@@ -85,6 +86,12 @@ def main() -> None:
     # Load a subset of CIFAR-10 to simulate the local data partition
     (x_train, y_train), (x_test, y_test) = load_partition(args.partition)
 
+    if args.toy:
+        (x_train, y_train), (x_test, y_test) = (x_train[:10], y_train[:10]), (
+            x_test[:10],
+            y_test[:10],
+        )
```
Comment (adam-narozniak) [inline examples/advanced_tensorflow/client.py:93]:
```suggestion
        x_train, y_train = x_train[:10], y_train[:10]
        x_test, y_test =     x_test[:10], y_test[:10]
```

Comment (adam-narozniak) [reply examples/advanced_tensorflow/client.py:93]: Just due to our formatting, it looks better

**Review by adam-narozniak — APPROVED** [review]
LGTM!

**Review by tanertopal — APPROVED** [review]

**Review by tanertopal — APPROVED** [review]

**Review by tanertopal — APPROVED** [review]

**Pull request merged**